demos/demos-v2.yaml

```diff
@@ -147,6 +147,25 @@ demos:
       cpu: 6800m
       memory: 15822Mi
       pvc: 28Gi
+  trino-subsea-data:
+    description: TODO
+    documentation: TODO
```

Comment:
Reminder to add description and docs :)

Reply:
I'll ask @stefanigel if we want to put it in the docs too or what. Will bring it up tomorrow in our Gaia-X Meeting.

Reply:
works for me

Reply:
Please remove the documentation field, it's optional ;)
https://github.com/stackabletech/stackable-cockpit/blob/3e2ed32c3ce0303c18463957c45012ef92419240/rust/stackable-cockpit/src/platform/demo/spec.rs#L64-L66